packages/migration/src/migrations/v0028_video3p.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function video3p({ pages, ...rest }) {
+  return {
+    pages: pages.map(reducePage),
+    ...rest,
+  };
+}
+
+function reducePage({ elements, ...rest }) {
+  return {
+    elements: elements.map(updateElement),
+    ...rest,
+  };
+}
+
+function updateElement(element) {
+  if (element.type === 'video' && element.resource && element.resource?.id) {
+    const idResource = element.resource?.id;
+    const idOrigin = idResource?.toString().split(':')?.[0];
+    const isCoverrMedia = idOrigin === 'media/coverr';
```

Comment:
Nit: we can probably do `const isCoverrMedia = idResource?.startsWith('media/coverr')`

Reply:
This is a copy and paste.